src/Microsoft.PowerShell.Commands.Utility/resources/NewObjectStrings.resx

```diff
@@ -138,6 +138,9 @@
   <data name="CannotInstantiateWinRTType" xml:space="preserve">
     <value>Creating instances of attribute and delegated Windows RT types is not supported.</value>
   </data>
+  <data name="CannotInstantiateBoxedByRefLikeType" xml:space="preserve">
+    <value>Cannot create instances of the ByRef-like type "{0}". ByRef-like types cannot be boxed and should be used only on stack.</value>
```

Comment:
I'd remove the mention of "stack" - seems no useful information for script writer.
```
ByRef-like types cannot be used in PowerShell Core scripts.
```

Reply:
Will update the message to be `ByRef-like types are not supported in PowerShell`.